src/@primer/gatsby-theme-doctocat/components/hero.js

```diff
@@ -0,0 +1,22 @@
+import loadable from '@loadable/component'
+import {Box, Heading} from '@primer/components'
+import React from 'react'
+import {Container} from '@primer/gatsby-theme-doctocat'
+import heroIllustration from '../../../hero-illustration.svg'
+
+const HeroAnimation = loadable(() => import('../../../hero-animation'), {
+  fallback: <img src={heroIllustration} alt="Interface Guidelines hero" width="100%" />
+})
+
+export default function Hero() {
+  return (
+    <Box bg="black" p={5}>
```

Comment:
`Container` already has some horizontal padding so we only need to set `py`.

```suggestion
    <Box bg="black" py={6}>
```